src/geom/rectangle/GetPoints.js

```diff
@@ -19,8 +19,8 @@ var Perimeter = require('./Perimeter');
  * @generic {Phaser.Geom.Point[]} O - [out,$return]
  *
  * @param {Phaser.Geom.Rectangle} rectangle - The Rectangle object to get the points from.
- * @param {number} step - Step between points. Used to calculate the number of points to return when quantity is falsy. Ignored if quantity is positive.
- * @param {integer} quantity - The number of evenly spaced points from the rectangles perimeter to return. If falsy, step param will be used to calculate the number of points.
+ * @param {number} step - Step between points. Used to calculate the number of points to return when quantity is falsey. Ignored if quantity is positive.
```

Comment:
I don't have a particularly strong personal preference between these two forms, but this repository does seem to, and it doesn't appear to have a mix of languages w/ distinct preferences (I've run into repositories which have that dichotomy).